cli-plugins/manager/manager_windows.go

```diff
@@ -0,0 +1,11 @@
+package manager
+
+import (
+	"os"
+	"path/filepath"
+)
+
+var defaultPluginDirs = []string{
+	filepath.Join(os.Getenv("USERPROFILE"), `.docker\cli-plugins`),
```

Comment:
There's already a [homedir](https://github.com/moby/moby/tree/master/pkg/homedir) package in `moby/moby`, I don't know if it's worth using it. WDYT @thaJeztah ? I also spotted that there will be a new [os.UserHomeDir](https://tip.golang.org/doc/go1.12#os) function in `golang-1.12`. I don't know yet if the moby `homedir` package will be dropped or patched with the new standard function.

Reply:
The code which computes `~/.docker/config.json` uses the moby `homedir` function and I think this should therefore do the same -- will change.

Reply:
Speaking of `~/.docker/config.json`, looking at the code I think perhaps this ought to obey `$DOCKER_CONFIG` too, which probably motivates pushing this down to somewhere around `cli/config/config.go` and using the same code.